_common/MarkdownHost.php

```diff
@@ -16,6 +16,10 @@ function ShowMenu() {
       return $this->showMenu;
     }
 
+    function PageDescription() {
+      return $this->pagedescription;
```

Comment:
I don't know PHP but it seems this could be uninitalised as ln 92 is has an IF. This inturn will effects https://github.com/keymanapp/keyman.com/blob/1e9c090f14b795eb8aeb21704aec348d2f361b13/_includes/includes/md/mdhost.php#L17

Reply:
good point. I've updated to return an empty string if pagedescription undefined.